docs/website/docs/general-usage/http/rest-client.md

```diff
@@ -311,7 +311,7 @@ When working with APIs that use non-standard pagination schemes, or when you nee
 
 - `update_request(request: Request) -> None`: Before making the next API call in `RESTClient.paginate` method, `update_request` is used to modify the request with the necessary parameters to fetch the next page (based on the current state of the paginator). For example, you can add query parameters to the request, or modify the URL.
 
-#### Example: creating a query parameter paginator
+#### Example 1: creating a query parameter paginator
```

Comment:
NIT cosmetics
```suggestion
#### Example 1: Creating a query parameter paginator
```

Reply:
We're using the sentence case for all headings, so I think it should start with a lowercase after ":". Or not?